Tasks/ServiceFabricPowerShellV1/task.loc.json

```diff
@@ -3,6 +3,7 @@
   "name": "ServiceFabricPowerShell",
   "friendlyName": "ms-resource:loc.friendlyName",
   "description": "ms-resource:loc.description",
+  "helpUrl": "https://go.microsoft.com/fwlink/?LinkID=841538",
```

Comment:
You need to update the versions of Service Fabric Powershell and Service Fabric Compose deploy as well

Reply:
The function Trace-WarningIfCertificateNotPresentInLocalCertStore is used only in ServiceFabric Deploy task as of now. These are just loc changes that were missed earlier.

Reply:
done